src/server/Gun.server.lua

```diff
@@ -0,0 +1,84 @@
+local Players = game:GetService("Players")
+local ReplicatedStorage = game:GetService("ReplicatedStorage")
+local Teams = game:GetService("Teams")
+local TweenService = game:GetService("TweenService")
+
+local convert = ReplicatedStorage:WaitForChild("Convert")
```

Comment:
WaitForChild() shouldn't be necessary on the server since the contents are already stored there.